### PR TITLE
refactor(nuxt): add link to the changelog on the community releases tab

### PR DESCRIPTION
**Pull request by bitbytebit1**
### 🔗 Linked issue
#22533

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
We should add a link to this page

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/5.community/7.changelog.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Discover the latest Nuxt 3 updates.
 
 # Releases
 
-Discover the latest Nuxt updates.
+[Discover the latest Nuxt updates.](https://nuxt.com/docs/community/changelog)
```
Comment (danielroe) [inline docs/5.community/7.changelog.md:9]: This is adding a link to itself. Is there a reason you think this is needed?

Comment (Barbapapazes) [reply docs/5.community/7.changelog.md:9]: We don't have source code but a Nuxt redirect is suspected here.

1. Go on this Google search page and click on the third link , https://www.google.com/search?q=nuxt+changelog&rlz=1C1ONGR_frFR1041FR1041&oq=nuxt+changelog&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIICAEQABgWGB4yCAgCEAAYFhgeMgYIAxBFGDwyBggEEEUYPDIGCAUQRRg80gEIMjQ5NmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8

**Comment by danielroe** [comment]
Thank you for this! I've directly pushed a fix to `nuxt/nuxt.com` (still private - sorry!) via a `routeRule`.

**Comment by Barbapapazes** [comment]
> Thank you for this! I've directly pushed a fix to `nuxt/nuxt.com` (still private - sorry!) via a `routeRule`.

Thanks!

**Pull request closed without merging**